README.md

```diff
@@ -89,11 +87,27 @@ You can run the build only for deploying KIE Maven artifacts using:
 
     mvn clean install -P !all,kie-artifacts
 
+You can build and run the Docker images using: 
+
+    mvn clean install -DskipTests
+
+  This will create two images:
+* kiegroup/business-central-wildfly23
+* kiegroup/kie-server-wildfly23
+
+now you can run the created Docker image starting a container
+
+    docker run -p 8082:8080 -p 8002:8001 -i -t --name <name> <imageName>:<tag>
+    i.e. docker run -p 8082:8080 -p 8002:8001 -i -t --name business-central kiegroup/business-central-wildfly23:7.63.0-SNAPSHOT
```

Comment:
```suggestion
    i.e. docker run -p 8082:8080 -p 8002:8001 -i -t --name business-central jboss-kie/business-central-wildfly23:7.63.0-SNAPSHOT
```

Reply:
i believe it generates still jboss-kie/<image-name> instead of kiegroup/<image_name>

Reply:
all docker mages have instead jboss/ now kiegroup/ - would leave this, if not we have to change the names of all Docker images

Reply:
yes, ok, i haven't updated last 3 commits, so now i can see the change